decidim-admin/app/permissions/decidim/admin/permissions.rb

```diff
@@ -66,6 +72,10 @@ def permissions
 
       private
 
+      def user_valuator?
+        user && !user.admin? && Decidim::ParticipatoryProcessUserRole.exists?(user: , role: "valuator")
```

Comment:
This is the rubocop error to fix: 
```suggestion
        user && !user.admin? && Decidim::ParticipatoryProcessUserRole.exists?(user:, role: "valuator")
```